app/controllers/foo_controller.rb

```diff
@@ -0,0 +1,13 @@
+class FooController < ActionController::Base  
+  def new_user
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
```

Comment:
__[staging][rubocop]__
* Useless assignment to variable - `user`.
* Use the new Ruby 1.9 hash syntax.